client/state/reader/watermarks/actions.js

```diff
@@ -14,7 +14,7 @@ import { READER_VIEW_STREAM } from 'state/action-types';
  * @param {String} streamId - stream being viewed
  * @returns {Object} action object for dispatch
  */
-export const viewStream = ( { mark, streamId } ) => {
+export const viewStream = ( { mark = Date.now(), streamId } ) => {
```

Comment:
As @blowery mentioned in Slack, we probably want to use the latest post's publish time here rather than `Date.now()`.

Reply:
done